geomstats/visualization/stiefel_manifold.py

```diff
@@ -0,0 +1,450 @@
+"""Stiefel manifold module."""
+import matplotlib.pyplot as plt
+import numpy as np
+
+import geomstats.backend as gs
+from geomstats.geometry.hypersphere import Hypersphere
+from geomstats.visualization import Sphere
+
+# %%
+"""
+    Consider <n=3, p=1> Stiefel Manifold.
+    Dimension is 2, which can be represented as sphere.
+"""
+
+
+class StiefelSphere:
+    """
+    Class used to plot something in Stiefel shape space of 2D triangles.
+
+    offering a 3D visualization of Kendall shape space of order
+    (3,1), and its related objects.
+
+    Attributes
+    ----------
+      ax : NoneType
+      Axis of the stiefel object
+
+    References
+    ----------
+    Absil, Pierre-Antoine. "Stiefel Manifolds and their Applications." (2009)
+    """
+
+    def __init__(self) -> None:
+        """Initialize the attributes."""
+        self.points = []
+        self.ax = None
+        self.set_ax()
+
+    def set_ax(self, ax=None):
+        """Set axis of Stiefel Sphere.
+
+        Parameters
+        ----------
+        ax : NoneType
+            Axis of the stiefel object
+
+        Returns
+        -------
+        None
+        """
+        if ax is None:
+            ax = plt.subplot(111, projection="3d")
+
+        ax_s = 0.5
+        plt.setp(
+            ax,
+            xlim=(-ax_s, ax_s),
+            ylim=(-ax_s, ax_s),
+            zlim=(-ax_s, ax_s),
+            xlabel="X",
+            ylabel="Y",
+            zlabel="Z",
+        )
+        self.ax = ax
+
+    def set_view(self, elev=60.0, azim=0.0):
```

Comment:
This seems to be copy-pasted from the visualization of Kendall shape spaces, which makes sense.
Instead of copy-pasting code, import the class from the kendall visualization and call it (wrap it)